boards/common/e104-bt50xxa-tb/Makefile.default

```diff
@@ -0,0 +1,2 @@
+# Default modules for e104-bt50xxa-tb boards
+DEFAULT_MODULE += board_software_reset
```

Comment:
This is indeed a better idea than extending `makefiles/default_modules.pk` :-)

Reply:
Oh, I see the mechanism for `Makefile.default` was already in place. I totally missed that. Good to learn about it - I already have a use case for it :-)

Reply:
No, I just introduced it in https://github.com/RIOT-OS/RIOT/pull/15835/commits/4d133e9889c8105aea79dc834d75616064124d5f

Reply:
I had a rough night :rofl: Thanks for adding this, this really is a clean solution!